SWLOR.Game.Server/Feature/AbilityDefinition/TwoHanded/CrossCutAbilityDefinition.cs

```diff
@@ -75,7 +75,10 @@ private static void ImpactAction(uint activator, uint target, int level, Locatio
                 defenderStat, 
                 0);
             ApplyEffectToObject(DurationType.Instant, EffectDamage(damage, DamageType.Slashing), target);
-            ApplyEffectToObject(DurationType.Temporary, EffectACDecrease(acLoss), target, 60f);
+
+            RemoveEffectByTag(target, "CROSS_CUT");
+            var eBreach = TagEffect(EffectACDecrease(acLoss), "CROSS_CUT");
```

Comment:
nit: don't prefix variables with a letter to represent the type. Just call this breach or effect